e2e/e2e_client_server.py

```diff
@@ -0,0 +1,41 @@
+
+import argparse
+import utils.CallBacks
+from utils.BluetoothHandler import BluetoothHandler
+
+def main():
+    parser = argparse.ArgumentParser()
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument("-i", help="flag indicating if should initiate as a server",
```

Comment:
```suggestion
    group.add_argument("--server", "-s", help="flag indicating if it should initiate as a server",
```

Perhaps we can have a "server" or a "client" command line arg like

```
python e2e_client_server.py --server
python e2e_client_server.py --client MAC_ADDRESS
```